cl/search/models.py

```diff
@@ -49,6 +49,17 @@
 HYPERSCAN_TOKENIZER = HyperscanTokenizer(cache_dir=".hyperscan")
 
 
+class SearchQuery(models.Model):
+    date_created = models.DateTimeField(auto_now_add=True, db_index=True)
+    date_modified = models.DateTimeField(auto_now=True, db_index=True)
+    get_params = models.CharField(max_length=255)
+    query_time_ms = models.IntegerField()
+    hit_cache = models.BooleanField()
+
+    def str(self):
+        return f"Query: {self.get_params} at {self.date_created}"
```

Comment:
Getting models exactly right is important because DB migrations are annoying, so I have a few comments here:

1. We can use `AbstractDateTimeModel` to automatically add the date-created and date_modified fields.
2. We'll want full descriptions of each field, as in other models.
3. In postgresql, you don't get a performance boost by doing max_length, and I can promise you that some queries will be really long. I'd remove that so that long queries don't crash things.
4. We'll want to link these queries to users, so we'll need a FK to them (there are lots of examples).
5. The documentation isn't (yet) clear about this, annoyingly, but `db_index` is considered the old way of adding an index. These days, they should go into the `Meta` class.